wp-parsely-2.5/README.md

```diff
@@ -1,8 +1,8 @@
 # Parse.ly
 
-Stable tag: 2.5.1  
+Stable tag: 2.5.2  
 Requires at least: 4.0  
-Tested up to: 5.6  
+Tested up to: 5.7  
```

Comment:
This probably needs a bump to 5.8.1 :)

Reply:
We'll bump it on the next Parse.ly release